integration_tests/testsuite/deploy_app.py

```diff
@@ -17,15 +17,19 @@
 import json
 import logging
 import os
-from retrying import retry
 import subprocess
 import sys
+from retrying import retry
+
+import test_util
+
+DEPLOYED_VERSION = None
 
 
 def _cleanup(appdir):
     try:
         os.remove(os.path.join(appdir, 'Dockerfile'))
-    except:
+    except Exception:
```

Comment:
this is just a fancy way of cleaning up the Dockerfile if it exists. if it doesn't, an exception happened in the `deploy_app` function, which gets logged.